packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -143,12 +145,27 @@ export const Reports = () => {
                   "ReportsConfigWidget:New"
                 )}
               </Button>
-              <div className="search-bar-container" data-testid="search-bar">
-                <SearchBar
-                  searchValue={searchValue}
-                  setSearchValue={setSearchValue}
-                  disabled={isLoading}
-                />
+              <IconButton
+                title={ReportsConfigWidget.localization.getLocalizedString(
+                  "ReportsConfigWidget:UpdateDatasets"
+                )}
+                onClick={async () => {
```

Comment:
Move out into `useCallback` and make it depend on `selectedReports`, `jobStartEvent`, and `bulkExtractor`.

Reply:
`onClick` function moved into `updateDatasets` callback.

Reply:
Looks good